### DISCUSSION TITLE
text-wrap: option to enable wrap

### DISCUSSION THREAD

**Discussion by technoogies** [KevinBatdorf/code-block-pro#325] [Ideas]
There are times when I add a single line of code that may be long and having an option to rwap text would be handy.

My patch for now.
`/* -[ Code Block Pro ]- */
.wp-block-kevinbatdorf-code-block-pro:not(.code-block-pro-editor):not(.cbp-has-line-numbers):not(.cbp-see-more-loaded) pre code .line {
    text-wrap: wrap !important;
}
/* -[ END Code Block Pro ]- */
`
Regards,
~ Cory C.

**Comment by KevinBatdorf** [comment]
I have to work on it some more since right now line wrapping would break line highlights and a couple other things. I appreciate the feedback though.
